src/CMakeLists.txt

```diff
@@ -1,9 +1,13 @@
 cmake_minimum_required (VERSION 3.8)
 
+# A interface to reuse common properties.
+add_library ("${PROJECT_NAME}_interface" INTERFACE)
```

Comment:
Especially useful for header test, as I do not need to build the libraries. I only need the includes.
Also useful when you build multiple libraries.